### PR TITLE
Add docs on Voila environment variables

### PR DESCRIPTION
**Pull request by akarve**
# TODO
* [x] Test `pd.read_csv(pkg["data.csv"].get_bytes())`

<del>
- [ ] Unit tests
- [ ] Automated tests (e.g. Preflight)
- [ ] Documentation
    - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
    - [ ] JavaScript: basic explanation and screenshot of new features
- [ ] [Changelog](CHANGELOG.md) entry
</del>

### CODE REVIEW

File: docs/Catalog/VisualizationDashboards.md [hunk 1]
```diff
@@ -228,6 +228,32 @@ in Linux containers that have network access but do not have access to persisten
...
+* `QUILT_PKG_BUCKET`
+* `QUILT_PKG_NAME`
+* `QUILT_PKG_TOP_HASH`
+
+You can access these variables in Python and browse the package:
+```python
+import pandas as pd
+import quilt3 as q3
+
+import os
```
Comment (sir-sigurd) [inline docs/Catalog/VisualizationDashboards.md:245]:
```suggestion
import os

import pandas as pd
import quilt3 as q3
```

File: docs/Catalog/VisualizationDashboards.md [hunk 2]
```diff
@@ -228,6 +228,32 @@ in Linux containers that have network access but do not have access to persisten
...
+
+bucket = os.environ.get("QUILT_PKG_BUCKET")
+handle = os.environ.get("QUILT_PKG_NAME")
+top_hash = os.environ.get("QUILT_PKG_TOP_HASH")
+
+pkg = q3.browse(handle, bucket, top_hash=top_hash)
+# Read data.csv from the current package from Voila
+df = pd.read_csv(pkg["data.csv"].get_bytes())
```
Comment (sir-sigurd) [inline docs/Catalog/VisualizationDashboards.md:253]:
```suggestion
df = pd.read_csv(io.BytesIO(pkg["data.csv"].get_bytes()))
```

Comment (sir-sigurd) [reply docs/Catalog/VisualizationDashboards.md:253]:
```
import pandas
import quilt3

pkg = quilt3.Package.browse('akarve/smartreports', registry='s3://searchminimal')
pandas.read_csv(pkg["data/ns.csv"].get_bytes())
```
fails with
```
OSError: Expected file path name or file-like object, got <class 'bytes'> type
```

File: docs/Catalog/VisualizationDashboards.md [hunk 3]
```diff
@@ -228,6 +228,33 @@ in Linux containers that have network access but do not have access to persisten
...
+import pandas as pd
+import quilt3 as q3
+
+bucket = os.environ.get("QUILT_PKG_BUCKET")
+handle = os.environ.get("QUILT_PKG_NAME")
+top_hash = os.environ.get("QUILT_PKG_TOP_HASH")
+
+pkg = q3.browse(handle, bucket, top_hash=top_hash)
+# Read data.csv from the current package from Voila
+df = pd.read_csv(io.BytesIO(pkg["data.csv"].get_bytes()))
```
Comment (akarve) [inline docs/Catalog/VisualizationDashboards.md:254]:
```suggestion
df = pkg["data.csv"]()
```

File: docs/Catalog/VisualizationDashboards.md [hunk 4]
```diff
@@ -228,6 +228,33 @@ in Linux containers that have network access but do not have access to persisten
...
+import pandas as pd
+import quilt3 as q3
+
+bucket = os.environ.get("QUILT_PKG_BUCKET")
+handle = os.environ.get("QUILT_PKG_NAME")
+top_hash = os.environ.get("QUILT_PKG_TOP_HASH")
+
+pkg = q3.browse(handle, bucket, top_hash=top_hash)
+# Read data.csv from the current package from Voila
+df = pkg["data.csv"]()
```
Comment (akarve) [inline docs/Catalog/VisualizationDashboards.md:254]:
```suggestion
df = pkg["data.csv"].deserialize()
```

File: docs/Catalog/VisualizationDashboards.md [hunk 5]
```diff
@@ -228,6 +228,33 @@ in Linux containers that have network access but do not have access to persisten
 storage. The catalog users's AWS credentials are passed to Jupyter kernel as
 [environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-list).
 
+## Environment variables
+When you have a Voila dashboard inside of a Quilt package, you may wish to reference files
```
Comment (sir-sigurd) [inline docs/Catalog/VisualizationDashboards.md:232]: Might make sense to mention that it doesn't work under BUCKET tab.

File: docs/Catalog/VisualizationDashboards.md [hunk 6]
```diff
@@ -228,6 +228,33 @@ in Linux containers that have network access but do not have access to persisten
...
+
+import pandas as pd
+import quilt3 as q3
+
+bucket = os.environ.get("QUILT_PKG_BUCKET")
+handle = os.environ.get("QUILT_PKG_NAME")
+top_hash = os.environ.get("QUILT_PKG_TOP_HASH")
+
+pkg = q3.browse(handle, bucket, top_hash=top_hash)
+# Read data.csv from the current package from Voila
```
Comment (sir-sigurd) [inline docs/Catalog/VisualizationDashboards.md:253]:
```suggestion
# Read data.csv from the current package revision.
```

**Review by sir-sigurd — APPROVED** [review]

**Pull request merged**